test/rtl/basic.mlir

```diff
@@ -2,44 +2,44 @@
 
 // CHECK-LABEL: func @test1(%arg0: i3) -> i50 {
 func @test1(%arg0: i3) -> i50 {
-  // CHECK-NEXT:    %c42_i12 = rtl.constant(42 : i12) : i12
```

Comment:
Not opposed to this, but random comment: the %cr42_i12 names are stable, because they are generated from the structure of the operation.  They don't depend on the order within the function.